routes/mods.py

```diff
@@ -136,8 +136,6 @@ def dict_all(models):
         if playtesters is not None:
             for playtester in playtesters:
                 seen = set()
-                unique_id = []
```

Comment:
Still hasn't been changed in other places.

Reply:
I removed that though in accordance to what you told me